Makefile

```diff
@@ -1,18 +0,0 @@
-SRC = $(wildcard src/*.js)
```

Comment:
Makefile should've not been removed, I use it all the time. What was the reason behind it?